app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -48,7 +48,8 @@ def repackage_message
     end
 
     rescue_from RestClient::Exception do |e|
```

Comment:
There's another controller - candlepin dynflow proxy controller - that may need the same treatment, but I haven't looked at the code for a while to know for sure

Reply:
`candlepin dynflow proxy controller` seems not need this change. 

Do you mean this one?https://github.com/Katello/katello/blob/be9af1d1c8071a29a940737bb26eaf2daef80fc9/app/controllers/katello/api/registry/registry_proxies_controller.rb#L27

Reply:
I meant https://github.com/Katello/katello/blob/master/app/controllers/katello/api/rhsm/candlepin_dynflow_proxy_controller.rb but you're right - no change needed there!